generators/environment/templates/outrigger/outrigger.yml

```diff
@@ -0,0 +1,90 @@
+##
+# This is an Outrigger configuration file used by rig project commands.
+##
+
+version: 1.0
+
+# This is prepended to the $PATH for any commands referenced in the scripts section.
+bin: ./bin:./node_modules/.bin:./vendor/bin
+
+# This is not currently used but we include it for completeness.
+project: <%= projectName %>
+
+# Project Scripts
+# These can be run via 'rig project run:<key>'
+# If you specify an alias, you can run 'rig project <alias>'
+scripts:
+
+  run:
+    description: Run one of the build container services such as 'cli' or 'drush'.
+    run:
+      - docker-compose -f $CWD/build.yml run --rm
+
+  setup:
+    alias: setup
+    description: Run the end-to-end repository initialization and site install script.
+    run:
+      - rig project run:welcome
+      - start.sh
+
+  up:
+    alias: up
+    description: Start up operational docker containers and filesystem sync.
+    run:
+      - rig project sync:start
+      - docker-compose up -d
+
+  stop:
+    alias: stop
+    description: Halt operational containers and filesystem sync.
+    run:
+      - docker-compose stop
+      - rig project sync:stop
+
+  build:
+    alias: build
+    description: Run the end-to-end build process.
+    run:
+      - docker-compose -f build.yml run --rm grunt
+
+  theme:
+    alias: theme
+    description: Compile all theme assets, activate watch, and start up pattern-lab.
+    run:
+      - docker-compose -f build.yml run --rm theme
+
+  install:
+    alias: install
+    description: Install the Drupal site.
+    run:
+      - docker-compose -f build.yml run --rm grunt install
+
+  ps:
+    description: Get a list of all containers associated with this project. Unlike docker-compose ps, this will include all containers even for different configuration files.
+    run:
+      - docker ps -a --filter "label=outrigger.project=<%= projectName %>"
+
+  all-stop:
+    description: Halt all containers associated with this project.
+    run:
+      - rig project run:ls -q | xargs docker stop
+
+  logs:
+    alias: logs
+    description: Stream the logs produced by your Docker project containers. To limit output add '--tail=20'.
+    run:
+      - docker-compose logs -ft
+
+  welcome:
+    alias: tour
+    description: Codebase orientation.
+    run:
+      - echo 'Welcome to the Project!'
+      - echo 'Please review the README.md, CONTRIBUTING.md, TODOS.md, and the docs/ folder.'
+      - echo 'All custom code is under src/'
+      - echo 'To get started, run "rig project setup"
+
+# This controls configuration for the `project sync:start` command.
+sync:
+  # This is the name of the external volume to use.
+  volume: project-sync
```

Comment:
this should be `<%= projectName %>-sync` instead of project-sync

In reality it will just discover this from the `docker-compose.yml` file, but it certainly doesn't hurt to specify it here, especially if we are driving it off the same code above that is rendering the configs.  We will need to update `sync:start` when we move to `docker-compose.yml` and `docker-compose.override.yml`